de.fu_berlin.inf.dpp/test/stf/de/fu_berlin/inf/dpp/stf/test/Constants.java

```diff
@@ -56,11 +56,11 @@
     public static final String CLS2_SUFFIX = "MyClass2.java";
     public static final String CLS3_SUFFIX = "MyClass3.java";
     /* content path */
-    public static final String CP1 = "test/resources/stf/" + CLS1 + ".java";
-    public static final String CP2 = "test/resources/stf/" + CLS2 + ".java";
-    public static final String CP3 = "test/resources/stf/" + CLS3 + ".java";
+    public static final String CP1 = "test/resources/stf/" + CLS1 + ".java.txt";
```

Comment:
I had to rename/move the files, because eclipse (using the gradle plugin) tried to compile the files and always fails.

Reply:
Wait ? There is a build config file in the plugin that clearly described what to compile, what to compile not and what to include. Why is Gradle compiling ?

Reply:
Gradle ignores these files, because I defined the corresponding directory `test/stf` as "resource" directory. The problem is how the gradle definitions are interpreted by buildship (eclipse gradle plugin). This issue appears only in eclipse (not in intellij or gradle only).